feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -0,0 +1,140 @@
+*   IS-IS routes advertised from ATE:port3 must have one next hop.
+*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+
+
+## AFT-1.1.1: AFT Base Link Down scenario 1
+
+Bring down the link between ATE:port2 and DUT:port2
+
+Verifications
+
+*   BGP routes advertised from ATE:port1,port2 must have 1 nexthop.
+*   IS-IS routes advertised from ATE:port3 must have one next hop.
+*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+
+## AFT-1.1.2: AFT Base Link Down scenario 2
+
+Bring down both links between ATE:port1,port2 and DUT:port1,port2
+
+Verifications
+
+*   BGP routes advertised from ATE:port1,port2 must be removed from RIB,FIB of the DUT, query results nil.
+*   IS-IS routes advertised from ATE:port3 must have one next hop.
+*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+
+
+## AFT-1.1.3: AFT Base Link Up scenario 1
+
+Bring up link between ATE:port1 and DUT:port1
+
+Verifications
+
+*   BGP routes advertised from ATE:port1,port2 must have one next hop.
+*   IS-IS routes advertised from ATE:port3 must have one next hop.
+*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+
+
+## AFT-1.1.4: AFT Base Link Up scenario 2
+
+Bring up both link between ATE:port1,port2 and DUT:port1,port2
+
+Verifications
+
+*   BGP routes advertised from ATE:port1,port2 must have 2 next hops.
+*   IS-IS routes advertised from ATE:port3 must have one next hop.
+*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+
+
+## OpenConfig Path and RPC Coverage
+
+The below yaml defines the OC paths intended to be covered by this test.  OC paths used for test setup are not listed here.
+
+```yaml
+paths:
+  ## Config Paths ##
+
+  ## State Paths ##
+ 
+  /network-instances/network-instance/afts/ethernet/mac-entry/state/next-hop-group:
+  /network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/next-hop-group:
+  /network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/origin-protocol:
+  /network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/prefix:
+  /network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/next-hop-group:
+  /network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/origin-protocol:
+  /network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/prefix:
+  /network-instances/network-instance/afts/aft-summaries/ipv4-unicast/protocols/protocol/state/origin-protocol:
+  /network-instances/network-instance/afts/aft-summaries/ipv4-unicast/protocols/protocol/state/counters/aft-entries:
+  /network-instances/network-instance/afts/aft-summaries/ipv6-unicast/protocols/protocol/state/origin-protocol:
+  /network-instances/network-instance/afts/aft-summaries/ipv6-unicast/protocols/protocol/state/counters/aft-entries:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/id:
```

Comment:
it is nedded.

Reply:
It is not needed, as mentioned.

`/network-instances/network-instance/afts/next-hop-groups/next-hop-group/id` is not the path one should read for state.

`/network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/id` holds the read info for this path, and it exists elsewhere in the list.

Can you remove all paths that are not `state`?

Reply:
A good approach is probably we should test for

1. gnmi.Subscribe requests using the gnmi streaming options (on_change, sample, etc) and the specific paths we subscribe during operations.
2. Presence of the specific leaf nodes we are depending on.  (because the subscribe could be at a container level, but the test validates the subscriptionReponse contents at the leaf level)